docs/modules/publisher-dmmigration.md

```diff
@@ -0,0 +1,141 @@
+| **GetConfiguredEndpoints**       | -                                                            | List<EndpointUrl>                                            | Yes                        |
+| **GetConfiguredNodesOnEndpoint** | EndpointUrl                                                  | EndpointUrl, List< OpcNodeOnEndpointModel >    where OpcNodeOnEndpointModel contains:    Id ExpandedNodeId OpcSamplingInterval OpcPublishingInterval DisplayName HeartbeatInterval SkipFirst | Yes                       |
+| **GetDiagnosticInfo**            | -                                                            | DiagnosticInfoMethodResponseModel                            | Yes                        |
+| **GetDiagnosticLog**             | -                                                            | MissedMessageCount, LogMessageCount, List<Log>               | No                      |
+| **GetDiagnosticStartupLog**      | -                                                            | MissedMessageCount, LogMessageCount, List<Log>               | No                      |
+| **ExitApplication**              | SecondsTillExit (optional)                                   | StatusCode, List<StatusResponse>                             | No                      |
+| **GetInfo**                      | -                                                            | GetInfoMethodResponseModel >>    VersionMajor  VersionMinor  VersionPatch  SemanticVersion  InformationalVersion OS  OSArchitecture  FrameworkDescription | No                      |
+
+A [sample configuration application](https://github.com/Azure-Samples/iot-edge-opc-publisher-nodeconfiguration) as well as an [sample application for reading diagnostic information](https://github.com/Azure-Samples/iot-edge-opc-publisher-diagnostics) are provided from OPC Publisher open-source, leveraging this interface.
+
+_Hint: The samples applications are not actively maintained and may be outdated._
+
+## Direct Methods of version 2.8.2
+
+There are some direct methods which are inherited from 2.5.x in addition to new ones.
+Inherited direct methods from 2.5.x:
+  - PublishNodes_V1
+  - UnpublishNodes_V1
+  - UnpublishAllNodes_V1
+  - GetConfiguredEndpoints_V1
+  - GetConfiguredNodesOnEndpoints_V1
+  - GetDiagnosticInfo_V1
+
+New direct methods:
+  - AddOrUpdateEndpoints_V1
+
+Note: `_V1` suffix is not required.
+
+### Terminologies
+
+The definitions of the important terms used are described below:
+
+- **DataSet** is a group of NodeIds within an OPC UA Server to be published with the same publishing interval.
+
+- **DataSetWriter** has one DataSet and contains the elements required to successfully establish a connection to the OPC UA Server.
+
+- **DatSetGroup** is used to group several DataSetWriters for a specific OPC UA server.
```

Comment:
@cristipogacean, isn't `DatSetWriterGroup` the correct term instead of `DatSetGroup` ?

Reply:
yes, it's DatSetWriterGroup